requirements/common.txt

```diff
@@ -1,6 +1,6 @@
 asgi-redis==1.4.3
 commonmark==0.5.4
-django==1.10.2
+django==1.11.6
```

Comment:
Please change it to `django==1.11`

Reply:
@RishabhJain2018 What will be the difference then? I think if don't write anything after the third then also it will fetch the latest version?! Please tell me!

Reply:
No, it will only download the version specified by you.

Reply:
@RishabhJain2018 Oh. Okay! Thanks